hooks/useRealm.tsx

```diff
@@ -27,11 +27,14 @@ import {
   switchboardPluginsPks,
   pythPluginsPks,
   gatewayPluginsPks,
+  heliumVsrPluginsPks,
 } from './useVotingPlugins'
 import useGatewayPluginStore from '../GatewayPlugin/store/gatewayPluginStore'
 import useSwitchboardPluginStore from 'SwitchboardVotePlugin/store/switchboardStore'
+import useHeliumVsrStore from 'HeliumVotePlugin/hooks/useHeliumVsrStore'
 import { BN } from '@coral-xyz/anchor'
 import { PublicKey } from '@solana/web3.js'
+import { useVsrMode } from './useVsrMode'
```

Comment:
to be clear, my intention was for `useVsrMode` to be used by components, rather than passing it through `useRealm` (since I don't like the way useRealm is just a mega-hook used for all sorts of disconnected data). But this is still step in right direction and I'm not going to bug you further about it. Thanks!